packages/zoe/src/zoeService/zoe.js

```diff
@@ -168,6 +170,24 @@ const makeDurableZoeKit = ({
     });
   };
 
+  const ZoeConfigI = M.interface('ZoeConfigFacet', {
+    updateZcfBundleId: M.call(M.string()).returns(),
+  });
+
+  const zoeConfigFacet = prepareExo(zoeBaggage, 'ZoeConfigFacet', ZoeConfigI, {
+    updateZcfBundleId(bundleId) {
+      E.when(
+        getZcfBundleCap({ id: bundleId }, vatAdminSvc),
+        bundleCap => {
+          zcfBundleCap = bundleCap;
+        },
+        e => {
+          console.warn(`unable to update ZCF Bundle: `, e);
```

Comment:
why only warn? This seems severe to me

Reply:
Zoe will continue to operate.  The request probably had the wrong format of bundleID/bundleCap/etc.  I'm amenable if you think it should be severe because the caller didn't get what they wanted.

Reply:
Yeah, something failed in this case. It's at least `console.error` and a siren.

If throwing doesn't break anything other than the requested operation (which already failed) then please throw.